src/qt/walletmodel.cpp

```diff
@@ -727,8 +727,14 @@ void WalletModel::listCoins(std::map<QString, std::vector<COutput> >& mapCoins)
         }
 
         CTxDestination address;
-        if(!out.fSpendable || !ExtractDestination(cout.tx->vout[cout.i].scriptPubKey, address))
+        if(cout.tx->IsZerocoinMint() || cout.tx->IsSigmaMint()){
+            mapCoins[QString::fromStdString("(mint)")].push_back(out);
```

Comment:
Is it possible to separate `(sigma mint) ` and `(zerocoin mint)`? I think it should be better. How do you think?

Reply:
Do you mean have only Sigma mint? As they are already separate now

Reply:
ok, I forgot we disable zerocoin now, have only 'mint' is make sense.